hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1203,11 +1203,41 @@ message UpdateGetS3SecretRequest {
     required string awsSecret = 2;
 }
 
+message OMNodeInfo {
+    required string nodeId = 1;
+    required string hostAddress = 2;
+    required uint32 rpcPort = 3;
+    required uint32 ratisPort = 4;
+    required string httpAddr = 5;
+    required string httpsAddr = 6;
+    optional string serviceId = 7;
+}
+
+enum BootstrapErrorCode {
+    RATIS_NOT_ENABLED = 1;
+    LEADER_UNDETERMINED = 2;
+    RATIS_BOOTSTRAP_ERROR = 3;
+}
+
+message BootstrapRequest {
```

Comment:
Can we call this AddOMRequest, as this is an admin command submitted to add new OM's to the cluster.

Reply:
Done.